crates/codegen/ebnf/src/builder.rs

```diff
@@ -177,11 +193,11 @@ impl Builder {
         self.add_entry(name, Terminal::No, Inlined::No);
 
         let mut expression = Expression::new_sequence(vec![
-            Self::build_ref(reference),
+            Self::build_ref(Some(BuiltInLabel::Item.as_ref()), reference),
```

Comment:
> A minor concern is the readability of the resulting EBNF, for sequences in particular. Adding the labels introduces noise to the spec document already. But for separated sequences, the resulting lines almost never fit into the enclosing box in the HTML, which forces horizontal scrolling.

Given the very common `Item (Sep Item)*` pattern, and that it is the same everywhere, I think it is fine for now. WDYT?